### ISSUE TITLE
Romance of the Three Kingdoms II visual noise

### ISSUE THREAD

**Issue by kpthomas85** [agg23/openfpga-SNES#18]
For Romance of the Three Kingdoms II, there is a horizontal line across the top that does not disappear, starting in load up screen and remaining throughout gameplay. This exact rom does not display this line on Super NT or SD2snes. 

![A5FD07D6-5266-4E72-8F55-77D2FFC6D048](https://user-images.githubusercontent.com/113795911/190870546-a3838dd1-cd28-42a0-ae10-6764d83d682e.jpeg)

**Comment by naveedster** [comment]
Not seeing this issue on a real 1chip snes, with Sony PVM monitor. 
![A3D7915F-7ED9-47A2-BF6B-614EDCCA797F](https://user-images.githubusercontent.com/80052032/190871782-bba6f36a-d30d-48c0-99b2-0f4cac05a735.jpeg)

**Comment by agg23** [comment]
I'm curious if this occurs on the MiSTer core. It's clearly in the overscan area, so people are less likely to notice, but it does appear to be unique to the core. It's also possible that this game does something different with the video that I am not properly accounting for.

**Comment by kpthomas85** [comment]
Yeah seems to only affect the pocket, didn’t see it on my snes jr with HDRV
cables to my crt either. I don’t have a mister so I unfortunately cannot
test that. I don’t suppose there’s a way to crop the overscan on pocket
(yet).

**Comment by naveedster** [comment]
Just to clarify, I even went into my monitors service menu and adjusted the vertical center, and vertical size (stretching of the video signal) and made sure that it wasn’t hidden by the overscan like it would be in a game like Mario for NES.

**Comment by naveedster** [comment]
What revision of SNES Hardware is the mister core based on? Maybe that is why?

**Comment by multi-vitamin** [comment]
The same symptoms appeared in ys3 and ys4.
However, since it is a translated rom, it may not appear in the original.

**Comment by Dubon77** [comment]
Not seeing this on Mister
![3650726C-1033-4827-B50F-3A7E9C2B51A9](https://user-images.githubusercontent.com/57742978/190896593-072fa520-00cf-4735-be48-6958fee4ab6f.jpeg)

**Comment by multi-vitamin** [comment]
If you play a game with black background like final fantasy 5,
The noise line disappears.

Perhaps the noise line is masked by other games or any data.

**Comment by SegaSnatcher** [comment]
What is the resolution of this game?  Maybe a scaling bug because of an odd resolution it's using?

**Comment by SegaSnatcher** [comment]
Ahh,  MiSTer is reporting a resolution of 256x239 for that game where most other games are 256x224.

**Comment by SegaSnatcher** [comment]
> ys3

Yep ys3 and lV also use the weird 256x239 resolution as this game.   So there is definitely an issue with that specific resolution.

**Comment by kpthomas85** [comment]
Nice! Well done
